gnome-image-installer/pages/diskimage/gis-diskimage-page.c

```diff
@@ -511,18 +517,24 @@ gis_diskimage_page_mount (GisPage *page)
       UDisksObject *object = UDISKS_OBJECT (l->data);
       UDisksBlock *block = udisks_object_peek_block (object);
       UDisksFilesystem *fs = NULL;
+      gboolean use_uuid = FALSE;
+      gboolean is_live = gis_store_is_live_install ();
       const gchar *const*mounts = NULL;
       const gchar *dev = NULL;
+      const gchar *uuid = gis_store_get_image_uuid ();
 
       if (block == NULL)
         continue;
 
       dev = udisks_block_get_preferred_device (block);
 
-      if (!g_str_equal (label, udisks_block_get_id_label (block)))
+      if (is_live && g_str_equal (uuid, udisks_block_get_id_uuid (block)))
+        use_uuid = TRUE;
+
+      if (!g_str_equal (label, udisks_block_get_id_label (block)) && !use_uuid)
         continue;
 
-      if (udisks_block_get_hint_ignore (block))
+      if (udisks_block_get_hint_ignore (block) && !use_uuid)
```

Comment:
> Keep that logic, but only when the host device is read-write (there's no danger of overwriting your Endless image on a read-only device!)

Probably this is not going to work with HD emulation where the partition is still RW. Probably I'll go with:
> Adjust dracut/image-boot/eos-image-boot-generator to not set UDISKS_IGNORE in the iso9660 case.